storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2514,6 +2636,22 @@ static bool xtrabackup_read_info(char *filename) {
   } else if (xb_server_version < 80029) {
     cfg_version = IB_EXPORT_CFG_VERSION_V6;
   }
+  /* skip start_time, end_time, lock_time, binlog_pos, innodb_from_lsn,
+   * innodb_to_lsn, partial, incremental, format, compressed, encrypt */
+  for (int i = 0; i < 12; i++) {
```

Comment:
I wish a better way of skipping parameter without the hardcoded 12. Not very important. Just a note.